server.py

```diff
@@ -0,0 +1,39 @@
+from flask import Flask, request, make_response, json
+#from find_melanoma import Melanoma
+import base64
+
+app = Flask(__name__)
+
+
+@app.route("/patient_classification", methods=['POST'])
+def patient_prediction():
```

Comment:
Don't forget docstrings!

Reply:
Thank you for the reminder, fixed it in the main.py.